src/ajax/check_hgvs_dialogue.php

```diff
@@ -35,17 +35,30 @@
 header('Content-type: text/javascript; charset=UTF-8');
 
 
+// Check whether all required input was given.
+if (!(isset($_REQUEST['var']) && isset($_REQUEST['action'])
+    && isset($_REQUEST['fieldName']) && isset($_REQUEST['refSeqInfo']))) {
+    // If any of these variables are missing, we cannot correctly
+    //  perform any checks, so we will exit the script.
+    exit;
+}
```

Comment:
This is fine, no need to change it, but just FYI: a perhaps more readable method is
```php
if (empty($_REQUEST['var']) || empty($_REQUEST['action'])
    || empty($_REQUEST['fieldName']) || empty($_REQUEST['refSeqInfo'])) {
```

Reply:
Much better, this has been implemented instead :)